tests/PhpPact/Standalone/ProviderVerifier/VerifierTest.php

```diff
@@ -80,20 +84,41 @@ protected function setUp(): void
             ['pactffi_verifier_set_provider_state', $this->handle, (string) $stateChangeUrl, $stateChangeTearDown, $stateChangeAsBody, null],
             ['pactffi_verifier_set_filter_info', $this->handle, $filterDescription, $filterState, $filterNoState, null],
             ['pactffi_verifier_set_verification_options', $this->handle, $disableSslVerification, $requestTimeout, null],
-            ['pactffi_verifier_set_publish_options', $this->handle, $providerVersion, $buildUrl, $this->isInstanceOf(CData::class), count($providerTags), $providerBranch, null],
-            ['pactffi_verifier_set_consumer_filters', $this->handle, $this->isInstanceOf(CData::class), count($filterConsumerNames), null],
+            [
+                'pactffi_verifier_set_publish_options',
+                $this->handle,
+                $providerVersion,
+                $buildUrl,
+                $hasProviderTags ? $this->isInstanceOf(CData::class) : null,
+                $hasProviderTags ? count($providerTags) : null,
```

Comment:
Just double checking, is the default value meant to be `null`, or should it be an empty array? The latter makes more sense to me.

Reply:
The default value is `null`, not empty array. Test will fail with this error if I change to empty array:


```
1) PhpPactTest\Standalone\ProviderVerifier\VerifierTest::testConstruct with data set #1 (false, false)
Failed asserting that null is identical to Array &0 [].
```

I explained why the value is `null` in this comment https://github.com/pact-foundation/pact-php/pull/527#discussion_r1518582953

Reply:
Thanks! After reading the comment, makes sense :)